packages/manager/modules/pci/src/projects/project/instances/instance/start/start.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class PciInstanceStartController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $translate,
+    Poller,
+    CucCloudMessage,
+    PciProjectsProjectInstanceService,
+    OvhApiCloudProjectInstance,
+  ) {
+    this.$q = $q;
+    this.$translate = $translate;
+    this.Poller = Poller;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciProjectsProjectInstanceService = PciProjectsProjectInstanceService;
+    this.OvhApiCloudProjectInstance = OvhApiCloudProjectInstance;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+  }
+
+  waitInstanceStart() {
+    const endPointUrl = `/cloud/project/${this.projectId}/instance/${this.instance.id}`;
+    return this.$q((resolve) => {
```

Comment:
Does `$q` needed here?

Reply:
Yes , I need it to resolve the start instance.